peps/pep-0729.rst

```diff
@@ -0,0 +1,413 @@
+PEP: 729
+Title: Typing governance process
+Author: Jelle Zijlstra <jelle.zijlstra@gmail.com>, Shantanu Jain <hauntsaninja at gmail.com>
+Discussions-To: https://discuss.python.org/t/proposed-new-typing-governance-process/34244
+Status: Draft
+Type: Process
+Topic: Typing
+Content-Type: text/x-rst
+Created: 19-Sep-2023
+Python-Version: 3.13
+Post-History: `20-Sep-2023 <https://discuss.python.org/t/proposed-new-typing-governance-process/34244>`__
+
+Abstract
+========
+
+This PEP proposes a new way to govern the Python type system: a council that is responsible
+for maintaining and developing the Python type system. The council will maintain a
+specification and conformance test suite and will initially be appointed by the Python Steering Council.
+
+Motivation
+==========
+
+The Python type system was created by :pep:`484`, now almost ten years ago. The type
+system is now widely used, and typing has become an important tool for writing
+good, maintainable Python code. Many changes have been made to the type system to cover
+more use cases and improve usability. Several type checkers have been created, each
+with their own strengths. The type annotation syntax has driven several major innovations
+in the Python ecosystem, such as the popular :mod:`dataclasses` module, runtime type
+checking and validation by packages such as Pydantic, and static compilation by tools
+such as mypyc.
+
+However, as the type system has grown, several interrelated problems with the current
+way to manage the type system have become apparent.
+
+PEPs are the only specification
+--------------------------------
+
+The Python type system was initially created by a PEP (:pep:`484`), and
+changes to the type system are still made by PEPs. The specification for
+the Python type system, to the extent there is one, consists of this series
+of PEPs. But PEPs aren't meant to be living documents
```

Comment:
[Indeed](https://peps.python.org/pep-0001/#pep-maintenance):

> In general, PEPs are no longer substantially modified after they have reached the Accepted, Final, Rejected or Superseded state. Once resolution is reached, a PEP is considered a historical document rather than a living specification.

But:

> Active (Informational and Process) PEPs may be updated over time to reflect changes to development practices and other details. 

So maybe? 

```suggestion
of PEPs. But Standard Track PEPs aren't meant to be living documents
```

Reply:
Sure, perhaps this? Or it's also fine as is.

```diff
-But PEPs aren't meant to be living documents
+But PEPs generally aren't meant to be living documents
``

Reply:
Maybe "these PEPs", referring back to the "series of PEPs" from the previous sentence, all of which are standards-track.

```suggestion
of PEPs. But these PEPs aren't meant to be living documents
```

Reply:
OK let's do it, but the correct spelling is Standard*s* Track:
```suggestion
of PEPs. But Standards Track PEPs aren't meant to be living documents
```